core/core.go

```diff
@@ -183,21 +215,35 @@ func validateName(name, password string) error {
 }
 
 // Init reads the records from disk from a given path
-func Init(path string) error {
+func Init(ca, hcKey, hcRoom, hcHost, roHost string) error {
```

Comment:
Why was this changed to "ca" from "path"? It still represents the location of the vault on disk.